.circleci/config.yml

```diff
@@ -18,12 +18,37 @@ jobs:
       - run:
           name: "Setup dbt"
           command: |
+            sudo apt install libsasl2-dev
             python3 -m venv venv
             . venv/bin/activate
             pip install --upgrade pip setuptools
             pip install -r integration_tests/requirements.txt
             mkdir -p ~/.dbt
             cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+      # - run:
+      #     name: "Run Tests - Postgres"
+      #     command: |
+      #       . venv/bin/activate
+      #       echo `pwd`
+      #       cd integration_tests
+      #       dbt deps
+      #       dbt seed --target postgres --full-refresh
+      #       dbt run --vars '{jira_using_sprints: false, jira_using_components: false}' --target postgres --full-refresh
+      #       dbt run --target postgres --full-refresh
+      #       dbt run --target postgres
+      #       dbt test --target postgres
+      # - run:
+      #     name: "Run Tests - Spark"
+      #     command: |
+      #       . venv/bin/activate
+      #       echo `pwd`
+      #       cd integration_tests
+      #       dbt deps
+      #       dbt seed --target spark --full-refresh
+      #       dbt run --vars '{jira_using_sprints: false, jira_using_components: false}' --target spark --full-refresh
+      #       dbt run --target spark --full-refresh
+      #       dbt run --target spark
+      #       dbt test --target spark
```

Comment:
You should probably remove this before merging our else there will be merge conflicts with the PR you created that handles the postgres and spark compatibility.